autoscale/src/main/java/com/sequenceiq/periscope/controller/validation/AlertValidator.java

```diff
@@ -127,6 +128,25 @@ public void validateClusterLimit(AdjustmentType adjustmentType, Integer configur
         }
     }
 
+    public void validateScheduleWithStopStartAndDisable(Cluster cluster, DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
+        if (!autoscaleClusterRequest.getTimeAlertRequests().isEmpty() &&
+                (Boolean.TRUE.equals(autoscaleClusterRequest.getUseStopStartMechanism()) || Boolean.TRUE.equals(cluster.isStopStartScalingEnabled()))) {
+            if (Boolean.TRUE.equals(cluster.isStopStartScalingEnabled())) {
+                asClusterCommonService.setStopStartScalingState(cluster.getId(), Boolean.FALSE);
```

Comment:
Do we want to change the state? Why not just throw the exception?

Reply:
Updated this to just throw the exception